design/gdd.md

```diff
@@ -77,31 +77,31 @@ They talk over the dinner about what happened. [calvin](characters/calvin.md) tr
 
 As [rebeca](characters/rebeca.md) doesn't tell her more, [marjorie](characters/marjorie.md) wants to investigate. She needs her phone for that.
 
 [rebeca](characters/rebeca.md) sits on the couch, [marjorie](characters/marjorie.md)'s phone sticking out of one of her pockets. [marjorie](characters/marjorie.md) needs to walk into the kitchen and look through the hatch to get it in the right moment.
 
-[marjorie](characters/marjorie.md)'s SIM is gone, so she needs wifi. There actually is wifi in the house used by the detectives, but it's locked.
+[marjorie](characters/marjorie.md)'s SIM card is gone, so she needs wifi. There actually is Wi-Fi in the house used by the detectives, but it's locked.
 
-[marjorie](characters/marjorie.md) traces the wifi signal to a loose board in the upstairs hall. She gets a knife from the kitchen and pries open the board to find the router. She turns the router around to finds a password. It isn't the right one, though. A paper in the junk in the kitchen shows how to decipher it.
+[marjorie](characters/marjorie.md) traces the Wi-Fi signal to a loose board in the upstairs hall. She gets a knife from the kitchen and pries open the board to find the router. She turns the router around to finds a password. It isn't the right one, though. A paper in the junk in the kitchen shows how to decipher it.
 
-She logs into the wifi and searches for "[jacob](characters/jacob.md) Grimes" to find that he had hung himself in her room 18 years ago and that [calvin](characters/calvin.md) was assigned to that case.
+She logs into the Wi-Fi and searches for "[jacob](characters/jacob.md) Grimes" to find that he had hung himself in her room 18 years ago and that [calvin](characters/calvin.md) was assigned to that case.
 
 #### Afternoon
 
 [brianna](characters/brianna.md) visits and [marjorie](characters/marjorie.md) asks her about [jacob](characters/jacob.md)'s story.
 
 #### Night
 
-[marjorie](characters/marjorie.md) wakes up from a sound, sits up in her bed and sees [igino](characters/igino.md) in the doorway, aiming a gun at her. He shoots and she wakes up and quickly sits up, gasping. It's early morning in day 3.
+[marjorie](characters/marjorie.md) wakes up from a sound, sits up in her bed and sees [igino](characters/igino.md) in the doorway, aiming a gun at her. He shoots and she wakes up and quickly sits up, gasping. It's early morning on day 3.
 
 ### Day 3
 
 #### Early morning
 
-[marjorie](characters/marjorie.md) sits in her bed. Her room is dark, only the dim light of the windows sheds some light into their side of the room.
+[marjorie](characters/marjorie.md) sits in her bed. Her room is dark, and only the dim light from the windows shed some light into the room.
```

Comment:
Shouldn't it be "shed**s**"?

Reply:
Of course. I was confused by "windows" there.